Firestore/src/FirestoreClient.php

```diff
@@ -399,7 +399,7 @@ public function documents(array $paths, array $options = [])
      * $query = $firestore->collectionGroup('users');
      * $querySnapshot = $query->documents();
      *
-     * echo sprintf('Found %d documents!', $querySnapshot->size());
+     * echo $querySnapshot->size() . ' documents found!';
```

Comment:
This change is needed because we dont interpret the `%` in snippets correctly, [ref](https://github.com/googleapis/google-cloud-php/blob/2af4e33d9dddb5f200eddd8827ea62a4b4162acc/Core/src/Testing/Reflection/DescriptionFactory.php#L99-L108)